<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#InIPAddress[state = &quot;0&quot;]{

	background-color: rgb(234, 0, 0);

}

#InIPAddress[state = &quot;1&quot;]{

	background-color: rgb(67,135,100);

}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Connection information</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_IPAddress">
           <property name="text">
            <string>IP Address</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="InIPAddress">
           <property name="inputMask">
            <string>000.000.000.000;_</string>
           </property>
           <property name="maxLength">
            <number>15</number>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
           <property name="cursorPosition">
            <number>0</number>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <property name="cursorMoveStyle">
            <enum>Qt::LogicalMoveStyle</enum>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
           <property name="state" stdset="0">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_Ports">
           <property name="text">
            <string>Port:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="spinBox">
           <property name="minimum">
            <number>1024</number>
           </property>
           <property name="maximum">
            <number>65535</number>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="btnConnect">
           <property name="text">
            <string>Connect</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="grpSendData">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>Send Data</string>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_Messgae">
           <property name="text">
            <string>Message</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_Message">
           <property name="text">
            <string/>
           </property>
           <property name="maxLength">
            <number>32767</number>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
           <property name="cursorPosition">
            <number>0</number>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <property name="cursorMoveStyle">
            <enum>Qt::LogicalMoveStyle</enum>
           </property>
           <property name="clearButtonEnabled">
            <bool>false</bool>
           </property>
           <property name="state" stdset="0">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="btnSend">
           <property name="text">
            <string>Send</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QListWidget" name="listConsole"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnConnect_2">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>604</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnConnect_2</sender>
   <signal>clicked()</signal>
   <receiver>listConsole</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>493</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>488</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
